### PR TITLE
Adding SFP entries for additional 10g ports

### PR DESCRIPTION
**Pull request by tjchadaga**
#### Why I did it
Fixes https://github.com/Azure/sonic-buildimage/issues/10038

#### How I did it
Added SFP entries for additional 10G ports added in https://github.com/Azure/sonic-buildimage/pull/9988

#### How to verify it
Load image and validate xcvrd stays in RUNNING state

#### Which release branch to backport (provide reason below if selected)

- [ ] 201811
- [ ] 201911
- [x] 202006
- [x] 202012
- [x] 202106
- [x] 202111

### CODE REVIEW

File: device/dell/x86_64-dell_s6100_c2538-r0/platform.json [hunk 1]
```diff
@@ -467,6 +467,12 @@
             {
                 "name": "QSFP+ or later"
             }
```
Comment (aravindmani-1) [inline device/dell/x86_64-dell_s6100_c2538-r0/platform.json:469]: Looks like comma is missing here.

File: device/dell/x86_64-dell_s6100_c2538-r0/platform.json [hunk 2]
```diff
@@ -467,6 +467,12 @@
...
+            {
+                "name": "QSFP+ or later"
+            }
```
Comment (aravindmani-1) [inline device/dell/x86_64-dell_s6100_c2538-r0/platform.json:472]: Looks like comma is missing here. Could you please check?.

**Review by aravindmani-1 — CHANGES_REQUESTED** [review]

File: device/dell/x86_64-dell_s6100_c2538-r0/platform.json (hunk 2 above)
Comment (aravindmani-1) [reply device/dell/x86_64-dell_s6100_c2538-r0/platform.json:472]: Port 65 and 66 are SFP+ ports. Not QSFP+ ports.

**Comment by aravindmani-1** [comment]
From the crash logs, it is crashing while retrieving _sfp_list[]. The changes you made doesn't seems to be resolving the crash. Could you execute "docker exec -it pmon bash", python3 /usr/local/bin/xcvrd and check whether the crash is seen?.

**Comment by prgeor** [comment]
@tjchadaga @aravindmani-1 can this be closed, now this [PR10062](https://github.com/Azure/sonic-buildimage/pull/10062) is raised.?

**Comment by tjchadaga** [comment]
Since the issue is resolved with https://github.com/Azure/sonic-buildimage/pull/10062, closing this PR

**Pull request closed without merging**